dali/operators/image/color/brightness_contrast.cc

```diff
@@ -24,42 +24,71 @@ using TheKernel = kernels::MultiplyAddCpu<Out, In, 3>;
 }  // namespace
 
 
-DALI_SCHEMA(BrightnessContrast)
-    .DocStr(R"code(Adjusts the brightness and contrast of the images based on the following
-formula::
-
-  out = brightness_shift * output_range + brightness * (grey + contrast * (in - grey))
-
-Where:
-
--	The output_range is 1 for float outputs or the maximum positive value for integral types.
--	Grey denotes the value of 0.5 for ``float``, 128 for ``uint8``, and 16384 for ``int16``, and so on.
+DALI_SCHEMA(Brightness)
+    .DocStr(R"code(Adjusts the brightness of the images based on the following formula::
+    out = brightness_shift * output_range + brightness * in
+Where output_range is 1 for float outputs or the maximum positive value for integral types.
 
 This operator can also change the type of data.)code")
     .NumInput(1)
     .NumOutput(1)
     .AddOptionalArg("brightness",
                     "Brightness mutliplier.",
-                    1.0f, true)
+                    kDefaultBrightness, true)
     .AddOptionalArg("brightness_shift", R"code(The brightness shift.
 
 For signed types, 1.0 represents the maximum positive value that can be represented by
 the type.)code",
-                    0.0f, true)
+                    kDefaultBrightnessShift, true)
+    .AddOptionalArg("dtype",
+                    R"code(Output data type.
+
+If not set, the input type is used.)code", DALI_NO_TYPE);
+
+DALI_SCHEMA(Contrast)
+    .DocStr(R"code(Adjusts the contrast of the images based on the following formula::
+    out = grey + contrast * (in - grey)
+Where:
+
+-	The output_range is 1 for float outputs or the maximum positive value for integral types.
+-	Grey denotes the value of 0.5 for ``float``, 128 for ``uint8``, and 16384 for ``int16``, and so on.
```

Comment:
Suggestion: rename grey to "midrange", "halfrange" or similar.

Reply:
Actually, there is an argument `contrast_center` that is used is this value, so I renamed `gray` to `contrast_center`. It has its own description so I removed this information from here.